cli/internal/publish/pluginui_test.go

```diff
@@ -0,0 +1,53 @@
+package publish
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestQuickContentType(t *testing.T) {
+	tests := []struct {
+		filename string
+		data     []byte
+		want     string
+	}{
+		{
+			filename: "octetstream.svg",
+			data:     []byte(`<svg xmlns="http://www.w3.org/2000/svg" width="2500" height="330" viewBox="0 0 719.713 94.943"><g fill-rule="evenodd" clip-rule="evenodd"><path d="M17.32,16.76l-.53.53-1.09-1.1a4.33,4.33,0,0,0,.53-.52Z" style="fill:#4285f4"/></g></svg>`),
```

Comment:
Can we embed these files instead to make it easier to read the tests?

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/19342/commits/a5579c5b3cfd96a495f3756dc62b78112ab7ffeb